migrations/versions/be0744a5679f_add_stripe_tables.py

```diff
@@ -0,0 +1,154 @@
+        sa.Column("stripe_price_id", sa.String(length=255), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("name"),
+        sa.UniqueConstraint("stripe_price_id"),
+        sa.UniqueConstraint("stripe_product_id"),
+    )
+    op.create_table(
+        "stripe_invoices",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("customer_id", sa.String(length=255), nullable=False),
+        sa.Column("invoice_id", sa.String(length=255), nullable=False),
+        sa.Column("hosted_invoice_url", sa.String(length=2048), nullable=False),
+        sa.Column("total", sa.Integer(), nullable=False),
+        sa.Column(
+            "status",
+            stripe_invoice_status_enum,
+            nullable=True,
+        ),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("tier_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["tier_id"],
+            ["tiers.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["user_id"],
+            ["users.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    with op.batch_alter_table("stripe_invoices", schema=None) as batch_op:
+        batch_op.create_index(
+            batch_op.f("ix_stripe_invoices_invoice_id"), ["invoice_id"], unique=True
+        )
+
+    with op.batch_alter_table("users", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("tier_id", sa.Integer(), nullable=True))
+        batch_op.add_column(sa.Column("stripe_customer_id", sa.String(length=255), nullable=True))
+        batch_op.add_column(
+            sa.Column("stripe_subscription_id", sa.String(length=255), nullable=True)
+        )
+        batch_op.add_column(
+            sa.Column("stripe_subscription_cancel_at_period_end", sa.Boolean, nullable=True)
+        )
+        batch_op.add_column(
+            sa.Column(
+                "stripe_subscription_status",
+                stripe_subscription_status_enum,
+                nullable=True,
+            ),
+        )
+        batch_op.add_column(
+            sa.Column(
+                "stripe_subscription_current_period_end",
+                sa.DateTime(timezone=True),
+                nullable=True,
+            ),
+        )
+        batch_op.add_column(
+            sa.Column(
+                "stripe_subscription_current_period_start",
+                sa.DateTime(timezone=True),
+                nullable=True,
+            ),
+        )
+        batch_op.create_index("idx_users_stripe_customer_id", ["stripe_customer_id"], unique=False)
+        batch_op.create_foreign_key("users_tier_id_fkey", "tiers", ["tier_id"], ["id"])
+
+
+def downgrade() -> None:
+    with op.batch_alter_table("users", schema=None) as batch_op:
+        batch_op.drop_constraint("users_tier_id_fkey", type_="foreignkey")
```

Comment:
This doesn't match the pattern in the metadata which is `"fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s"`. Having it differ would mean other migrations would drop/create it.

Reply:
Fixed in 40b1ca102ea381b325f06f6a1ec5ceb61c4228a2